koalas-requirements.txt

```diff
@@ -1,3 +1,4 @@
 pyspark>=3.0.0,<3.2.0
-koalas>=1.8.0,<=1.8.2
-pandas<1.4.0
+koalas>=1.8.0,<=1.8.2; python_version < '3.10'
```

Comment:
Since koalas is in maitenence mode, will we ever be able to support 3.10? If not, do we need a long term plan for dropping support for koalas?

Reply:
@tamargrey We have an issue to update to use the pyspark pandas API instead of Koalas longer term: https://github.com/alteryx/woodwork/issues/1263

Reply:
So is it OK to leave koalas for python3.10 here for now as it will be updated with #1263?

Reply:
@mingdavidqi Yes, I think it should be fine to limit the Python version here when using Koalas until #1263 is implemented.